### PR TITLE
[Test] Use @codeCoverageIgnore for deprecated class/method

### PR DESCRIPTION
**Pull request by samsonasik**
**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

**Review by paulbalandan — APPROVED** [review]
I think we can also use the ignoreDeprecatedCodeUnits key of the coverage key in phpunit.xml.dist in addition to these annotations.

**Comment by samsonasik** [comment]
I've added ignoreDeprecatedCodeUnits="true" to phpunit config https://github.com/codeigniter4/CodeIgniter4/pull/4623/commits/0eeb9f6e0611315199c4c2df2fd82aee9723c863

File: phpunit.xml.dist [hunk 1]
```diff
@@ -9,7 +9,8 @@
 		 colors="true"
 		 failOnRisky="true"
 		 failOnWarning="true"
-		 verbose="true">
+		 verbose="true"
+		 ignoreDeprecatedCodeUnits="true">
 
 	<coverage processUncoveredFiles="true">
```
Comment (paulbalandan) [inline phpunit.xml.dist:15]:
```suggestion
		 verbose="true">

	<coverage processUncoveredFiles="true" ignoreDeprecatedCodeUnits="true">
```
This should be on the `coverage` element per phpunit's XSD

Comment (samsonasik) [reply phpunit.xml.dist]: updated

**Review by paulbalandan — APPROVED** [review]

**Comment by samsonasik** [comment]
@paulbalandan thank you for the review.

**Pull request merged**